src/components/table/table.tsx

```diff
@@ -9,26 +10,32 @@ import TableContainer from '@mui/material/TableContainer'
 import TableHead from '@mui/material/TableHead'
 import TablePagination from '@mui/material/TablePagination'
 import TableRow from '@mui/material/TableRow'
+import TableSortLabel from '@mui/material/TableSortLabel'
+import Tooltip from '@mui/material/Tooltip'
 import Typography from '@mui/material/Typography'
 import React, { useState } from 'react'
 
 import { capitalizeFirstLetter } from '../helper'
-import type { TableData, TableHeader } from '../types'
+import Icon from '../icon/icon'
+import type { DataRow, TableData, TableHeader } from '../types'
 
 const paginationThreshold = 10
 
-export default function Table(props: TableData) {
-  const isPaginationEnabled = props.data.length > paginationThreshold
+type Order = 'asc' | 'desc'
```

Comment:
I think we should use a string enum instead. That will also make it easier for generating tooltip labels